### DISCUSSION TITLE
ScriptReference/BuildTargetGroup

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#60] [Notes]
# ScriptReference/BuildTargetGroup

https://docs.unity3d.com/ScriptReference/BuildTargetGroup.html

**Comment by vertxxyz** [comment]
See [`BuildPipeline.GetBuildTargetGroup(EditorUserBuildSettings.activeBuildTarget))`](EditorUserBuildSettings.activeBuildTarget), or, if using an [IPreprocessBuildWithReport](https://docs.unity3d.com/ScriptReference/Build.IPreprocessBuildWithReport.html): [`report.summary.platformGroup`](https://docs.unity3d.com/ScriptReference/Build.Reporting.BuildSummary-platformGroup.html).
